zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -194,8 +193,8 @@ object ServerSpec extends HttpRunnableSpec(8088) {
     }
 
   private val socketApp = SocketApp.message(socket) ++ SocketApp.protocol(SocketProtocol.handshakeTimeout(100 millis))
-  private val webSocketApp = Http.collect[Request] { case Method.GET -> !! / "subscriptions" =>
-    Response.socket(socketApp)
+  private val webSocketApp = Http.collectM[Request] { case Method.GET -> !! / "subscriptions" =>
+    ZIO(Response.socket(socketApp))
```

Comment:
use `Http.fromEffect` instead.

Reply:
This can't be done since we have another app composed after this if I use `Http.fromEffect`, all request will match this app and none will go to the next app